scipy/signal/signaltools.py

```diff
@@ -1724,32 +1724,42 @@ def cmplx_sort(p):
 
 
 def unique_roots(p, tol=1e-3, rtype='min'):
-    """
-    Determine unique roots and their multiplicities from a list of roots.
+    """Determine unique roots and their multiplicities from a list of roots.
 
     Parameters
     ----------
     p : array_like
         The list of roots.
     tol : float, optional
-        The tolerance for two roots to be considered equal. Default is 1e-3.
-    rtype : {'max', 'min, 'avg'}, optional
+        The tolerance for two roots to be considered equal in terms of
+        the distance between them. Default is 1e-3. Refer to Notes about
+        the details on roots grouping.
+    rtype : {'max', 'maximum', 'min', 'minimum', 'avg', 'mean'}, optional
         How to determine the returned root if multiple roots are within
         `tol` of each other.
 
-          - 'max': pick the maximum of those roots.
-          - 'min': pick the minimum of those roots.
-          - 'avg': take the average of those roots.
+          - 'max', 'maximum': pick the maximum of those roots
+          - 'min', 'minimum': pick the minimum of those roots
+          - 'avg', 'mean': take the average of those roots
+
+        Note that complex roots are compared first by the real part and then
+        by the imaginary part.
```

Comment:
This note seems outdated, it just uses Euclidean distance now right?

Reply:
I added this note. I wanted to clarify how minimum and maximum is computed for complex roots in case `rtype='min'/'max'`.

What do you think?

Reply:
It would be good to clarify that this is only relevant in `min/max` mode

Reply:
It's done.